identity/webapp/src/frontend/MyAccount/DeleteAccount.tsx

```diff
@@ -123,7 +124,7 @@ export const DeleteAccount: React.FC<ChangeDetailsModalContentProps> = ({
           </FieldMargin>
           <ButtonAlign>
             <ButtonSolid
-              isDangerous
```

Comment:
React novice here: is this equivalent to `isDangerous={true}`? And hence you're setting the danger colours?

Reply:
You got it https://reactjs.org/docs/jsx-in-depth.html#props-default-to-true